LICENSE

```diff
@@ -1,7 +1,31 @@
-Node's license follows:
+Node.js is licensed for use as follows:
 
-====
+"""
+Copyright Node.js contributors. All rights reserved.
```

Comment:
Same as https://github.com/nodejs/node/pull/3965/files#r45569502:

By making additive changes to the license, are we changing it? If so, it is valid? Have we consulted someone familiar with the topic on that matter?